packages/elements/src/react/common/form/index.tsx

```diff
@@ -1,3 +1,4 @@
+import { isWebAuthnAutofillSupported } from '@clerk/clerk-js/src/utils/passkeys';
```

Comment:
Same as in your https://github.com/clerk/javascript/pull/3472/files#r1621275247 comment